builtin/logical/pki/acme_state.go

```diff
@@ -139,8 +153,15 @@ func (a *ACMEState) ParseRequestParams(data *framework.FieldData) (*JWSCtx, map[
 		return nil, nil, fmt.Errorf("invalid or reused nonce")
```

Comment:
nit: Should be wrapped with a `ErrBadNonce`?

Reply:
Ah yeah, this was added before I did the error handling stuff.